src/main/java/org/apache/cassandra/service/PerTableSSTableInfo.java

```diff
@@ -0,0 +1,66 @@
+package org.apache.cassandra.service;
+
+import static com.sun.jmx.mbeanserver.MXBeanMappingFactory.DEFAULT;
+
+import java.io.InvalidObjectException;
+import java.util.List;
+
+import javax.management.openmbean.CompositeData;
+import javax.management.openmbean.OpenDataException;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import com.sun.jmx.mbeanserver.MXBeanMapping;
+
+@SuppressWarnings("restriction")
+@XmlRootElement
```

Comment:
What is this `XmlRootElement` annotation about?

Reply:
To enable automatic marshalling of object <-> json via java REST client.

Reply:
Obviously... I think I have forgotten the power of Java enterprise APIs!

Thanks for the clarification!

Reply:
Powerful apis they are... Powerful apis.